docs/apidiffs/1.35.0_vs_1.34.1/opentelemetry-exporter-zipkin.txt

```diff
@@ -0,0 +1,9 @@
+Comparing source compatibility of  against 
+***  MODIFIED CLASS: PUBLIC FINAL io.opentelemetry.exporter.zipkin.ZipkinSpanExporterBuilder  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	===  UNCHANGED METHOD: PUBLIC io.opentelemetry.exporter.zipkin.ZipkinSpanExporterBuilder setEncoder(zipkin2.codec.BytesEncoder<zipkin2.Span><zipkin2.Span>)
```

Comment:
any idea what the second `<zipkin2.Span>` is for? 🤔

Reply:
That's strange..